src/main/webapp/style.css

```diff
@@ -6,4 +6,8 @@ html {
     box-sizing: inherit;
     margin: 0;
     padding: 0;
+}
+
+body #map{
+    height: 400px;
```

Comment:
The div that hosts the map has a default 0 height. This is just to make it visible when the website loads.